### PR TITLE
Clarify output messages (#144)

### PR DESCRIPTION
**Pull request by Risen-phoenix**
All output and error messages within aurman have been clarified. User choice methods have also been updated to keep format as y/n.

### CODE REVIEW

**Comment by Risen-phoenix** [comment]
I've decided to close the request, as it doesn't seem you agree with the wording of any of the changes.

**Comment by polygamma** [comment]
Sorry, I just did not understand the reason for most of the changes to be honest. Is the english "too german" or are the descriptions too verbose? You changed the meaning of almost every text passage, which really irritated me.

**Comment by Morganamilo** [comment]
Many of the aurman messages are "I can tell this was written by someone who is not first language English but they are still clear and coherent". That's how I would describe it at least.

For example "acquire sudo failed" is a little off but I do understand it so not a big deal. This is one of the worst messages.

Then there's things like "AUR helper with almost pacman syntax" which like you said, add character and I agree.

I do not like most of the changes in this PR either though, This is just my opinion on the current messages.

**Comment by polygamma** [comment]
Thanks for your opinion @Morganamilo. I am always grateful for suggestions improving the wording used in aurman, but as I said: This PR mostly irritated me, because I do not get the reason for changing the meaning. Some of the "new" descriptions are really just wrong

**Comment by polygamma** [comment]
quite off topic, but if you want to laugh: https://github.com/polygamma/breakout/blob/master/src/breakout/Main.java

my first university project, also my first java project. german english and weird java code, but the game is really nice though. i am still playing it sometimes when I am bored.

**Comment by Morganamilo** [comment]
Oh god this brings back memories of helping my bf with his uni coursework.

Especially:

```java
    private static volatile int stateOfProgram = -2;
```

He got marked down for magic numbers :(

Then there's his lecturer's weird conventions that we both hated, like avoid breaks and continues as much as possible, avoid returns not at the end of functions.

And by the way you uploaded the compiled jar file to the repo ;) (although I know this is on purpose cause the lecturers don't want to bother compiling source)

**Comment by polygamma** [comment]
acm.jar is actually not the compiled game but the library we had to use, see: https://www-cs-faculty.stanford.edu/people/eroberts/jtf/

the whole java course was based on this stanford thing, which was weird in the beginning, but all in all I really liked it. 

regarding the magic number: yeah, that's really lazy designed, but if i remember it correctly, the whole "breakout project" was done over only 2 weeks, one fellow student made the graphics and I coded the game.

**Comment by Morganamilo** [comment]
Hey I still remember the java commands.

image

File: src/aurman/main.py [hunk 1]
```diff
@@ -674,10 +674,10 @@ def process(args):
             solutions, concrete_packages_to_install, upstream_system, not only_unfulfilled_deps, solution_way
         )
     except InvalidInput:
-        aurman_error("we could not find a solution")
+        aurman_error("dependency conflict")
         # if not --deep_search
         if only_unfulfilled_deps:
-            aurman_error("if you think that there should be one, rerun aurman with the --deep_search flag")
+            aurman_error("to ignore currently fulfilled dependencies, run aurman with --deep-search")
```
[the remark this answers is not in the document]
Comment (eli-schwartz) [reply src/aurman/main.py]: More problematically, this advises people to run a nonexistent `--deep-search` flag. Why was this string modified in ways that are just unambiguously incorrect?

**Comment by polygamma** [comment]
@Morganamilo, did you also play the three levels?

@Risen-phoenix, Why the downvote instead of answering?

**Comment by Morganamilo** [comment]
I played them. I don't have the patience to beat them all though. It's a nice little game for something made in two weeks.

**Comment by frealgagu** [comment]
I'll play it once I have my pc available, I'm writing from my phone right now.

**Pull request closed without merging**